kotlin-analysis/src/main/kotlin/org/jetbrains/dokka/analysis/AnalysisEnvironment.kt

```diff
@@ -496,8 +507,9 @@ class AnalysisEnvironment(val messageCollector: MessageCollector, val analysisPl
     fun addClasspath(paths: List<File>) {
         if (analysisPlatform == Platform.js) {
             configuration.addAll(JSConfigurationKeys.LIBRARIES, paths.map { it.absolutePath })
+        } else {
+            configuration.addJvmClasspathRoots(paths)
```

Comment:
Is it possible that a similar problem will arise if `paths` contains files from `Platform.native` or `Platform.common`?

Reply:
Possible, but it will depend on runners. I do not expect the similar problem here.
This PR solves only issues which are related with JS deps.